packages/zipkin-instrumentation-kafkajs/README.md

```diff
@@ -24,8 +24,8 @@ const kafka = instrumentKafkaJs(new Kafka({
   clientId: 'my-app',
   brokers: ['localhost:9092']
 }), {
-    tracer, //Your zipkin tracer instance
-    remoteServiceName : 'node-backend' //When using producer, the remote endpoint
+    tracer, // Your zipkin tracer instance
+    remoteServiceName : 'kaka' // This should be the symbolic name of the broker, not a consumer.
```

Comment:
is `kaka` correct?

Reply:
given that "kaka" in Russian means "poop", that's a very rhetorical question, although I'm sure Adrian likes Kafka and that's just a typo :D

Reply:
LOL actually I misspelled.. I meant caca or more specifically 💩